test/Fail/UnquoteUnbound.err

```diff
@@ -1,2 +1,8 @@
-UnquoteUnbound.agda:9.1-13.32: error: [GenericError]
-Unbound name: UnquoteUnbound.a-fresh-name
+UnquoteUnbound.agda:9.1-13.32: error: [NotInScope]
+Not in scope:
+  UnquoteUnbound.a-fresh-name
+  at UnquoteUnbound.agda:1.8-22
+    (did you mean
+       'Agda.Builtin.Reflection.freshName' or
+       'Common.Reflection.freshName' or
+       'freshName'?)
```

Comment:
Here I was wondering whether this is what the users of reflection want to see, or whether there should be a variant of NotInScope just for the reflection.

Reply:
Conceptually it is a very different error, so it wouldn't be wrong to use a separate error code.

Reply:
Agreed with Ulf.